var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -28,40 +31,53 @@ class PyElephant(PythonPackage):
     version("0.3.0", sha256="747251ccfb5820bdead6391411b5faf205b4ddf3ababaefe865f50b16540cfef")
 
     variant("docs", default=False, description="Install documentation dependencies")
-    variant("pandas", default=False, description="Build with pandas", when="@0.3.0:0.4.1")
     variant(
         "extras", default=False, description="Build with extras for GPFA, ASSET", when="@0.6.4:"
     )
 
     depends_on("py-setuptools", type="build")
-    depends_on("python@3.7:", type=("build", "run"), when="@0.11.0:")
+
+    depends_on("python@3.8:", type=("build", "run"), when="@master")
+    depends_on("python@3.8:", type=("build", "run"), when="@0.12.0:")
+    depends_on("python@3.7:", type=("build", "run"), when="@0.11.0:0.11.2")
+
+    depends_on("py-neo@0.10.0:", type=("build", "run"), when="@master")
+    depends_on("py-neo@0.10.0:", type=("build", "run"), when="@0.11.0:")
```

Comment:
```suggestion
    depends_on("py-neo@0.10:", type=("build", "run"), when="@0.11:")
```
Same for other deps, master is always newer than all stable releases

Reply:
Removed deps for master.